src/pages/PageApi.js

```diff
@@ -0,0 +1,51 @@
+class PageApi {
```

Comment:
naming: RequestApi to keep consistent with name change of pages folder to requests